cpu/stm32/periph/flashpage.c

```diff
@@ -118,8 +126,19 @@ static void _erase_page(void *page_addr)
     pn = (uint8_t)page;
 #endif
     CNTRL_REG &= ~FLASH_CR_PNB;
+#if defined(CPU_FAM_STM32F4) || defined(CPU_FAM_STM32F7)
```

Comment:
```suggestion
```

Reply:
Can't do this without defining a fallback macro for `FLASH_CR_SNB_4` for all families that don't implement this.